MobileBuy/build.gradle

```diff
@@ -21,6 +26,25 @@ allprojects {
     }
 }
 
+// snapshot of dependencies to use
```

Comment:
This allows the test app to control the versions of the deps that the android sdk is built with without needing to modify code in the android sd repo itself.